draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+disbursement mechanism is established in a future ZIP.
+
+**Decentralization**: The lockbox must be designed with the goal of future
+decentralization in mind. While the disbursement mechanism will be defined
+later, the lockbox should support the principles of decentralization to ensure
+future community involvement and project-specific funding.
+
+**Regulatory Considerations**: The creation of the lockbox should minimize
+regulatory risks by avoiding direct funding of specific organizations. The
+design should ensure compliance with applicable laws and regulations to support
+the long-term sustainability of the funding model.
+
+**Efficient Allocation Preparation**: The lockbox must be prepared to allocate
+resources efficiently once the disbursement mechanism is defined. This includes
+ensuring that funds are readily available for future projects and not tied up
+in organizational overhead.
+
+**Security of Funds**: The lockbox must implement robust security measures to
+protect against unauthorized access or misuse of funds. This includes ensuring
+the integrity of the future decentralized voting/signature mechanism.
+
+**Future Adaptability**: The lockbox design should be flexible and adaptable to
+accommodate the disbursement mechanism defined in a future ZIP. This ensures
+that the system can evolve based on feedback and new developments in the Zcash
+ecosystem.
+
+**Non-Direct Funding Model**: This ZIP recognizes there is strong community
+support for a non-direct funding model.  As such, ECC and ZF MUST collaborate
+with the Zcash community to research and explore the establishment of a
+non-direct funding model. The research should consider potential designs as
+well as possible legal and regulatory risks.
+
+**ZCG Slice**: The ZCG slice should encourage decentralization and support
+independent teams dedicated to Zcash. Major funding decisions should be based,
+to the extent feasible, on inputs from domain experts and pertinent
+stakeholders.
+
+**Value of ZEC**: In case the value of ZEC jumps, ZCG should not wastefully use
+excessive amounts of funds. Conversely, given market volatility and eventual
+halvings, it is desirable to create rainy-day reserves.
+
+**Funding Stream**: The ZCG funding stream should conform to the standards of
+ZIP 207 and ZIP 214.
+
+## Specification
+
+Consensus changes implied by this specification are applicable to the Zcash
+Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be
+applied to the Zcash Testnet for testing purposes.
+
+### Hybrid Deferred Dev Fund Allocation
+
+The Zcash Development Fund SHALL be enacted in Network Upgrade 6 and last for 2
+years. The Dev Fund would consist of 20% of the block subsidies, split into 2
+slices:
+
+* 60% to a deferred Dev Fund “lockbox,” which will be allocated in the future
+  once consensus is reached on fund allocation or a new non-direct funding
+  model is established and implemented.
```

Comment:
Process constraints and consensus constraints should be separate. This is a consensus constraint:
```suggestion
* 60% to a deferred Dev Fund “lockbox”.
```

The process constraint needs to be described in a separate subsection of the specification, in order for this ZIP to be semantically stable.